README.md

```diff
@@ -1,5 +1,16 @@
 # Dopamine
 
+**This branch contains an experimental, proof-of-concept implementation of
```

Comment:
What do you thing about explicitly writing 'WARNING: xxx'?

Reply:
Done.